src/db/nedb.js

```diff
@@ -74,7 +74,7 @@ class DBHelper {
     let fieldsObj;
```

Comment:
why don't we use reduce here?
```js
const fieldsObj = !_.isEmpty(fields) && fields.reduce((field, obj) => obj[field] = 1, {});
```

Reply:
can change this in a different PR. my purpose for this one is just to fix the tests. ran the linter so this change in this file showed up.